string object_id
bool picked_up
float32 x
float32 y
float32 z
float32 errno
